README.md

```diff
@@ -1,2 +1,6 @@
 # photo-gallery
 Pictures!
+
```

Comment:
This is fine for now, but remember that the TA's are going to need access to your PR's to make sure your process and practices are semantic.